content/blog/deploy-stateful-mongodb-application-on-kubernetes-clusters-in-hpe-greenlake-for-containers-1.md

```diff
@@ -0,0 +1,547 @@
+---
+title: Deploy Stateful MongoDB Application on Kubernetes Clusters in HPE
```

Comment:
Hi there! So, HPE brand now wants us to use sentence case for titles. So the title needs to change a little here. In addition, you should either say "applications" or "a stateful MongoDB application" 

So the title could look like either of these:
Deploy stateful MongoDB applications on Kubernetes clusters in HPE GreenLake for Containers
Deploy a stateful MongoDB application on Kubernetes clusters in HPE GreenLake for Containers

I recommend the first one myself...